client/blocks/login/two-factor-authentication/verification-code-form.jsx

```diff
@@ -143,9 +143,9 @@ class VerificationCodeForm extends Component {
 
 					<FormFieldset>
 						<FormLabel htmlFor="twoStepCode">{ labelText }</FormLabel>
-
+						// The use of `autoFocus` is intentional in this short flow.
```

Comment:
can't use `//` comments inside react like this
```suggestion
						{ // The use of `autoFocus` is intentional in this short flow. }
```

Reply:
hah, i see Don added the same comment just before me. Sorry for the dupe.